src/ctl.c

```diff
@@ -2107,6 +2114,8 @@ CTL_RO_CGEN(config_stats, stats_arenas_i_internal,
     stats_arenas_i(mib[2])->astats.internal, size_t)
 CTL_RO_CGEN(config_stats, stats_arenas_i_resident,
     stats_arenas_i(mib[2])->astats.resident, size_t)
+CTL_RO_CGEN(config_stats, stats_arenas_i_tcache_bytes,
```

Comment:
I think the first argument to the macro should be ```config_stats && config_tcache```, no that this mallctl endpoint returns ```ENOENT``` if tcache isn't built in.